python/seldon_core/wrapper.py

```diff
@@ -113,6 +114,10 @@ def __init__(self, user_model):
         self.user_model = user_model
 
     def Predict(self, request_grpc, context):
+        data_type = request_grpc.WhichOneof("data_oneof")
```

Comment:
If people use proto buffers I don't think they need to base64 encode their data as its a binary protoc? So I don't think we need this.

Reply:
@cliveseldon Reverted